crates/stackable-operator/src/yaml.rs

```diff
@@ -2,8 +2,18 @@
 use std::io::Write;
 
 use serde::ser;
+use snafu::{ResultExt, Snafu};
 
-use crate::error::OperatorResult;
+type Result<T, E = Error> = std::result::Result<T, E>;
+
+#[derive(Debug, Snafu)]
+pub enum Error {
+    #[snafu(display("failed to serialize YAML"))]
+    YamlSerialization { source: serde_yaml::Error },
```

Comment:
nitpick: Adjust variant name.

```suggestion
    SerializeYaml { source: serde_yaml::Error },
```

Reply:
Done in e7134b0